frontend/src/components/LandingPage/LandingPage.js

```diff
@@ -1,10 +1,17 @@
 import React from 'react'
-//Import components that you create here
+import Tool from './Tools/Tool';
+import Tools from './Tools/Tools'
 
 function LandingPage(){
+    //Creates various Tool objects that contain names and info about the various tools
+    var reactTool = new Tool("react", "React is a declarative, efficient, and flexible JavaScript library for building user interfaces\nUses a virtual DOM and does a diff of the real DOM to only rebuild the parts that need to be rebuilt\nIt lets you compose complex UIs from small and isolated pieces of code called “components”\nReact can be added to most existing HTML pages with no complicated tools or install requirements");
```

Comment:
What do you think about creating a constants file that has the info inside of it in object form so that our Landingpage doesnt get cluttered? You can use Katys pr as an example. Take a look at her [TeamMembers.js](https://github.com/Codubee/SpaceJunkTTH2-4/blob/CB-4-teamcomponent/frontend/src/components/LandingPage/TeamComponent/TeamMembers.js) file for an example.